spec/support/saml_idp_bad_destination_controller.rb.erb

```diff
@@ -0,0 +1,187 @@
+class SamlIdpController < SamlIdp::IdpController
```

Comment:
I think it might be easier to maintain this template if there's only one, and we extract a method to handle the destination swapping. I've [pushed an example to the repo](https://github.com/apokalipto/devise_saml_authenticatable/compare/usertesting-fix-infinite-loop...usertesting-fix-infinite-loop-adam). What do you think?